app/src/main/java/com/es/findsoccerplayers/Utils.java

```diff
@@ -82,6 +82,7 @@ public void onSuccess(Void aVoid) {
                 Log.w(tag, "Database successfully updated with new general match info");
             }
         });
+        return true;

```

Comment:
questo non serve molto, tipo ritorna sempre true.

Reply:
Mi serve perchè su create match se la funzione torna true mostra un toast di successo. E' da implementare la casistica in cui fallisce

Reply:
eh si, vabbeh lasciamo cosi per ora, metti un TODO